go/git/get.go

```diff
@@ -21,7 +21,7 @@ type ServerResponseRepo struct {
 	EncryptionVersion     int                           `json:"encryption_version"`
 	Nonce                 string                        `json:"nonce"`
 	KeyGeneration         keybase1.PerTeamKeyGeneration `json:"key_generation"`
-	LastModifyingUsername string                        `json:"last_writer_username"`
+	LastModifyingUID      keybase1.UID                  `json:"last_writer_uid"`
```

Comment:
Note that this field is already returned by the server: https://github.com/keybase/keybase/pull/1732/files#diff-620039165948b62e1b58f6640e2dc78eL878